.node-version

```diff
@@ -1 +0,0 @@
-18.16.0
```

Comment:
Why remove this?

Reply:
Because the `.nvmrc` file already exists! Having both conflicts node.

Reply:
`.node-version` exists because our repos support both `nodenv` and `nvm` (which uses `.nvmrc`).

What kind of conflicts are you experiencing?

Reply:
Having both files I used nvm by doing a `nvm install`. After the installation has finished successfully, running yarn gives me the following error.
`nodenv: version `18.16.0' is not installed `
It seemed so there was some conflict between `nvm` and `nodenv`.
But now I put the file again and the error seems gone. I'll add it again